demisto_sdk/commands/changelog/tests/changelog_test.py

```diff
@@ -0,0 +1,105 @@
+import os
+from pathlib import Path
+
+import pytest
+
+from demisto_sdk.commands.changelog import changelog
+from demisto_sdk.commands.changelog.changelog import Changelog
+from demisto_sdk.commands.changelog.changelog_obj import INITIAL_LOG
+from demisto_sdk.commands.common.handlers import YAML_Handler
+
+yaml = YAML_Handler()
+
+
+@pytest.fixture
+def changelog_mock():
+    return Changelog(pr_name="test", pr_number="12345")
+
+
+def test_is_release(changelog_mock):
+    """
+    Given:
+        - Changelog obj with some different `pr_name`
+    When:
+        - run the is_release method
+    Then:
+        - Ensure return True only if pr_name is in vX.X.X format
+    """
+    assert not changelog_mock.is_release()
+    changelog_mock.pr_name = ""
+    assert not changelog_mock.is_release()
+    changelog_mock.pr_name = "v1.10.2"
+    assert changelog_mock.is_release()
+    
+
+def test_is_log_folder_empty(tmpdir, changelog_mock: Changelog):
+    """
+    Given:
+        - Changelog obj and a temporary path with a .changelog folder
+    When:
+        - run the is_log_folder_empty method
+    Then:
+        - Ensure return True only if there is a file in the .changelog folder
+    """
+    folder_path = Path(tmpdir / ".changelog")
+    os.makedirs(folder_path)
+    changelog.CHANGELOG_FOLDER = folder_path
```

Comment:
changing a constant in a unit test is ill-advised, please mock it instead (here, and in other test cases)